keyboards/converter/ps2_usb/README.md

```diff
@@ -0,0 +1,78 @@
+PS/2 to USB keyboard converter
```

Comment:
Could you please rename this file as `readme.md`?

Reply:
yup

Reply:
This file still needs to be renamed.